src/v1/container_analysis_proto_list.json

```diff
@@ -1,3 +1,15 @@
 [
-  "../../protos/google/devtools/containeranalysis/v1/containeranalysis.proto"
-]
+  "../../protos/google/devtools/containeranalysis/v1/containeranalysis.proto",
+  "../../protos/grafeas/v1/attestation.proto",
+  "../../protos/grafeas/v1/build.proto",
+  "../../protos/grafeas/v1/common.proto",
+  "../../protos/grafeas/v1/cvss.proto",
+  "../../protos/grafeas/v1/deployment.proto",
+  "../../protos/grafeas/v1/discovery.proto",
+  "../../protos/grafeas/v1/grafeas.proto",
+  "../../protos/grafeas/v1/image.proto",
+  "../../protos/grafeas/v1/package.proto",
+  "../../protos/grafeas/v1/provenance.proto",
+  "../../protos/grafeas/v1/upgrade.proto",
+  "../../protos/grafeas/v1/vulnerability.proto"
+]
```

Comment:
nit: please add `\n` when you print this file.

Reply:
done, thank you!